### PR TITLE
[DEV-1521]: update webinars images

### PR DESCRIPTION
**Pull request by MikeAtUqido**
#### List of Changes

Update webinars images

#### Motivation and Context

The webinars images were low resolution

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by tommaso1 — APPROVED** [review]

File: apps/nextjs-website/src/_contents/webinars.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ export const webinars: readonly Webinar[] = [
       `<p>Esploriamo il nuovo modello di comunicazione su app IO, che consente nuovi scenari d’uso e nuove esperienze utente grazie alle tecniche di remotizzazione.
        </p> \n` +
       `<br /> \n` +
-      `<img style="height: auto;max-width: 100%" src='/images/webinar-io-remote-content.png' /> \n` +
+      `<img style="height: auto;max-width: 820px" src='/images/webinar-io-remote-content.png' /> \n` +
```
Comment (MarcoPonchia) [inline apps/nextjs-website/src/_contents/webinars.ts:58]: Without `100%` max-width on small viewport or device image overflow page view.
```suggestion
      `<img style="height: auto;width: 820px;max-width: 100%" src='/images/webinar-io-remote-content.png' /> \n` +
```

File: apps/nextjs-website/src/_contents/webinars.ts [hunk 2]
```diff
@@ -105,7 +105,7 @@ export const webinars: readonly Webinar[] = [
       `<p>Come semplificare ed efficientare la gestione delle multe con PagoPA? </br> In questo appuntamento saranno illustrati tutti i passaggi utili per la costruzione di un servizio pienamente digitale ed efficiente attraverso l’utilizzo sinergico di pagoPA, IO e SEND. Il webinar ha l’obiettivo infatti di fornire tutti i chiarimenti e gli strumenti utili per una corretta e virtuosa gestione digitale del servizio multe attraverso l’utilizzo integrato delle piattaforme di PagoPA.
        </p> \n` +
       `<br /> \n` +
-      `<img style="height: auto;max-width: 100%" src='/images/webinar-pago-pa-multe.jpg' /> \n` +
+      `<img style="height: auto;max-width: 820px" src='/images/webinar-pago-pa-multe.jpg' /> \n` +
```
Comment (MarcoPonchia) [inline apps/nextjs-website/src/_contents/webinars.ts:108]: Same here.
```suggestion
      `<img style="height: auto;width: 820px;max-width: 100%" src='/images/webinar-pago-pa-multe.jpg' /> \n` +
```

File: apps/nextjs-website/src/_contents/webinars.ts [hunk 3]
```diff
@@ -158,13 +158,13 @@ export const webinars: readonly Webinar[] = [
       `<h4 style="font-weight: 600; font-size: 24px;">Tutto quello che serve per integrarsi con le nuove API</h4> \n` +
       `<p>Durante il webinar, approfondiremo la creazione, l'invio di un servizio e la gestione di scenari comuni, come "Ho creato un servizio e lo vedo in stato di bozza. Cosa devo fare?". Partecipate a questo webinar per ottenere una comprensione approfondita e per imparare a gestire in modo efficace il ciclo di vita dei vostri servizi su IO. </p> \n` +
       `<br /> \n` +
-      `<img style="height: auto;max-width: 100%" src='/images/webinar-io-sm.png' /> \n` +
+      `<img style="height: auto;max-width: 820px" src='/images/webinar-io-sm.jpg' /> \n` +
```
Comment (MarcoPonchia) [inline apps/nextjs-website/src/_contents/webinars.ts:161]: Same here.
```suggestion
      `<img style="height: auto;width: 820px;max-width: 100%" src='/images/webinar-io-sm.jpg' /> \n` +
```

**Review by MarcoPonchia — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/_contents/webinars.ts (hunk 1 above)
Comment (MikeAtUqido) [reply apps/nextjs-website/src/_contents/webinars.ts]: I inverted width and max-width from your suggestion, the max-width should be set to `820px` or else on desktop viewports the image results too large

**Review by marcobottaro — APPROVED** [review]

**Review by MarcoPonchia — APPROVED** [review]

**Pull request merged**